litmus-portal/frontend/src/views/Observability/Overview/index.tsx

```diff
@@ -192,7 +192,13 @@ const Overview: React.FC = () => {
   let filteredDashboardListData;
   // Select the latest 3 dashboards
   if (monitoringDashboardCount > 0) {
-    filteredDashboardListData = dashboardListData?.ListDashboard.slice(-3);
+    filteredDashboardListData = dashboardListData?.ListDashboard.slice()
+      .sort(
+        (a, b) =>
+          (b.viewed_at as unknown as number) -
```

Comment:
directly as number was not working?

Reply:
It is coming as a string from the backend.. sort accepts number type, ref to typecast logic: https://stackoverflow.com/questions/53813188/how-can-i-cast-custom-type-to-primitive-type

Reply:
This can be used @vanshBhatia-A4k9 https://github.com/litmuschaos/litmus/blob/fdab91443197a2ff68697314c2973d4a41e89cdd/litmus-portal/frontend/src/utils/sort.ts#L10